source/sass/mozfest.scss

```diff
@@ -181,7 +181,10 @@ body.mozfest {
       }
     }
   }
-
+  // Overriding sticky nav z-index to show tito widget
```

Comment:
Hi @richbrennan! Unfortunately, it looks like the tito widget is adding its own CSS that we cannot override to push its position lower unless we remove the styles completely as shown here: https://ti.to/docs/api/widget#tito-widget-v2-form-styles

Instead, I reduced the z-index of the nav bar so the widget can be completely shown instead of covered by the nav bar. Please let me know if this works, thanks!

Reply:
This works for me, I'll hand over to Marc to try, thanks Daniel!